src/main/scripts/festerize/README.md

```diff
@@ -4,11 +4,11 @@ Uploads CSV files to the Fester IIIF manifest service for processing.
 
 Any collection rows found in the CSV are used to create a IIIF collection.
 
-Any work rows are used to expand or revise a previously created IIIF collection (corresponding to the collection that the work is a part of), as well as create a IIIF manifest corresponding to the work.
+Any work rows are used to expand or revise a previously created IIIF collection (corresponding to the collection that the work is a part of), as well as create a IIIF manifest corresponding to the work. A "work" is conceived of as a discrete object that should be access as an individual item. An example of a "work" is a book or a photograph. 
```

Comment:
_accessed_

`can be accessed` instead of should?

Reply:
Good edit!